Client/Frontend/AuthenticationManager/PasscodeEntryViewController.swift

```diff
@@ -121,3 +121,19 @@ extension PasscodeEntryViewController: PasscodeInputViewDelegate {
         passcodePane.codeInputView.resetCode()
     }
 }
+
+#if canImport(SwiftUI) && MOZ_CHANNEL_FENNEC
```

Comment:
That Preview stuff is not needed, just updating Strings in this PR is enough.
There are no plans to use SwiftUI for our project at the moment.